### PR TITLE
Scaffold with package for yesNoPage

### PR DESCRIPTION
**Pull request by liampan**
### Description of Work carried through

Added package definition to yes no pages

This will need to be done for any other of the pages we want to use scaffolds for. I suggest just using this PR to copy the changes for speed, as ive use this now to generate and check it works :D 

Brief description of what/why/how.

Ive also solved some of the other issues we had with scaffoled pages like using cacheConnector rather that session repo, so would be good to do this to all

### CODE REVIEW

File: test-utils/generators/PageGenerators.scala [hunk 1]
```diff
@@ -18,8 +18,8 @@ package generators
 
 import org.scalacheck.Arbitrary
 import pages._
-import pages.amends.{AmendAreYouSurePage, AmendDirectExportPlasticPackagingPage, AmendHumanMedicinePlasticPackagingPage, AmendImportedPlasticPackagingPage, AmendManufacturedPlasticPackagingPage, AmendRecycledPlasticPackagingPage}
```
Comment (liampan) [inline test-utils/generators/PageGenerators.scala:21]: this wont need done for future scaffolds, just made this catch all :D

File: test-utils/generators/UserAnswersGenerator.scala [hunk 2]
```diff
@@ -21,8 +21,8 @@ import org.scalacheck.Arbitrary.arbitrary
 import org.scalacheck.{Arbitrary, Gen}
 import org.scalatest.TryValues
 import pages._
-import pages.amends.{AmendAreYouSurePage, AmendDirectExportPlasticPackagingPage, AmendHumanMedicinePlasticPackagingPage, AmendImportedPlasticPackagingPage, AmendManufacturedPlasticPackagingPage, AmendRecycledPlasticPackagingPage}
```
Comment (liampan) [inline test-utils/generators/UserAnswersGenerator.scala:24]: this wont need done for future scaffolds, just made this catch all :D

File: .g8/yesNoPage/default.properties [hunk 3]
```diff
@@ -1,2 +1,3 @@
 description = Generates a controller and view for a page with a single yes / no question
+package = returns
```
Comment (liampan) [inline .g8/yesNoPage/default.properties:2]: this is the default, feels ok to use returns? if not feel free to overrule :)

**Review by mauro-fanton — APPROVED** [review]

**Review by C-Thomp — APPROVED** [review]
very useful 👍

File: .g8/yesNoPage/default.properties (hunk 3 above)
Comment (C-Thomp) [reply .g8/yesNoPage/default.properties:2]: Still have to type `returns` in response to the question, i think this just puts suggestion text next to the prompt

**Review by ajchilts — APPROVED** [review]

**Pull request merged**